MetricsDashboard.Core/Entities/TileEntity.cs

```diff
@@ -0,0 +1,16 @@
+﻿using MetricsDashboard.Contract.Enums;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace MetricsDashboard.Core.Entities
```

Comment:
Entities -> MongoEntities?

Reply:
Are there any other kind of entities? I think just entities would be sufficient.